ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -318,6 +318,23 @@ const LocalStandardDisks = ({ idPrefix, setIsFormValid, onAddErrorNotification }
         }
     ));
 
+    const discoveringDisksColumns = localDisksColumns.map(col => ({ ...col, sortable: false }));
+
+    const discoveringDisksRow = (
+        [
+            {
+                props: { colSpan: localDisksColumns.length },
+                title: <Skeleton screenreaderText={_("Discovering disks")} />
```

Comment:
Done as it is, the text will repeat for every skeleton row, I think. But that's not too important right now.